config/locales/en.yml

```diff
@@ -28,6 +28,24 @@ en:
     cohort_does_not_accept_capping: "Leave the '#/funded_place' field blank. It's only needed for participants starting NPQs from autumn 2024 onwards."
     cannot_change_funded_place: "You must void or claw back your declarations for this participant before being able to set '#/funded_place' to false"
 
+  declaration: &declaration
+    future_declaration_date: "The '#/declaration_date' value cannot be a future date. Check the date and try again."
+    mismatch_declaration_type_for_schedule: "The property '#/declaration_type' does not exist for this schedule."
```

Comment:
I think this should be under declaration type, as testing locally I'm getting a mismatch error?